solverdummy/README.md

```diff
@@ -0,0 +1,23 @@
+# MATLAB solverdummy
```

Comment:
Shouldn't the complete solverdummy folder be in `examples`?

Reply:
For the https://github.com/precice/python-bindings we also provide the solverdummies directly under `/solverdummies`.

I'm fine with leaving everything as it is, but for consistency putting the solverdummies into `examples/solverdummies` would make sense.

Reply:
Putting them into `examples` would be consistent with precice itself. So I think this should be the way to go for all bindings.